vm/http_client.go

```diff
@@ -0,0 +1,119 @@
+package vm
+
+import (
+	"errors"
+	"fmt"
+	gerrors "github.com/goby-lang/goby/vm/errors"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+// Instance methods --------------------------------------------------------
+func builtinHTTPClientInstanceMethods() []*BuiltinMethodObject {
+	goClient := http.DefaultClient
+
+	return []*BuiltinMethodObject{
+		{
+			// Sends a GET request to the target and returns the HTTP response as a string.
+			Name: "send",
+			Fn: func(receiver Object) builtinMethodBody {
+				return func(t *thread, args []Object, blockFrame *callFrame) Object {
+
+					if len(args) != 0 {
+						return t.vm.initErrorObject(gerrors.ArgumentError, "Expect 0 arguments. got=%v", strconv.Itoa(len(args)))
+
+					}
+
+					req := httpRequestClass.initializeInstance()
+
+					result := t.builtinMethodYield(blockFrame, req)
+
+					if err, ok := result.Target.(*Error); ok {
+						fmt.Printf("Error: %s", err.Message)
+						return err //a Error object
+					}
+
+					goReq, err := requestGobyToGo(req)
+					if err != nil {
+						return t.vm.initErrorObject(gerrors.ArgumentError, "Request object incomplete object %s", err)
```

Comment:
This message's grammar seems a little weird. @hachi8833 Can you take a took?

Reply:
I'll change it to "Request object incomplete %s", my mistake